rn/Teacher/src/modules/courses/CourseNavigation.js

```diff
@@ -106,6 +106,8 @@ export class CourseNavigation extends Component<CourseNavigationProps, any> {
         if (tab.id === 'pages') {
           const url = `/courses/${this.props.courseID}/pages`
           this.props.navigator.show(url)
+        } else if (tab.id === 'collaborations') {
+          this.props.navigator.showWebView(tab.full_url)
```

Comment:
should this just be `show` and the fact that it becomes a web view a detail of the route handler? Or is the full_url containing additional info that will get lost?

Reply:
Good call. Fixed.